lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.scss

```
@@ -194,3 +194,16 @@ $content-node-selector-thumbnail-width: 35px !default;
         }
     }
 }
+
+
+.adf-content-node-selector-content-list {
+    position: relative;
+
+    .adf-content-node-selector-content-list-empty {
+        & + adf-infinite-pagination {
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+        }
+    }
```

Minor, but this could be contracted a little:
```suggestion
    &-empty + adf-infinite-pagination {
        position: absolute;
        bottom: 0;
        width: 100%;
    }
```